### PR TITLE
restructured the whole components folder

### PR DESCRIPTION
**Pull request by leealex88**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
Provide a short description of the change you are proposing and the reasons for it.

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/components/AreasOfFocus/styles.css [hunk 1]
```diff
@@ -69,6 +69,19 @@
     border-radius: 50%;
 }
 
-.add-area-button {
-    
+.input-lights-add-button-container {
+    display: flex;
+}
+
+.area-input-sectio{
```
Comment (tayfuntayfun) [inline client/src/components/AreasOfFocus/styles.css:76]: Hi Ola, you are missing the last letter at section. Can you complete the word "section" please. Thanks a lot.

File: client/src/components/FormAreasOfFocus/index.js [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+				<div className="area-input-section">
+					<input
+						className="areas-input"
+						type="text"
+						name="area"
+						placeholder="eg. CSS, useEffect Hook, for loop..."
+						onChange={(e) =>
+							setAreaForm({
+								...areaForm,
+								message: e.target.value,
```
Comment (tayfuntayfun) [inline client/src/components/FormAreasOfFocus/index.js:41]: Hi Ola, can you apply the changes we discussed on the call today.

File: client/src/components/StudentCard/index.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
 /* eslint-disable jsx-a11y/no-static-element-interactions */
 /* eslint-disable react/prop-types */
```
Comment (tayfuntayfun) [inline client/src/components/StudentCard/index.js:2]: Great stuff Ola as always. One silly comment here; could it be possible for you delete eslint stuff before any PR.

**Review by tayfuntayfun — APPROVED** [review]
This is another sample of great work Ola. Just wanted to leave a couple of small comments. Thanks again.

File: client/src/components/FormAreasOfFocus/index.js [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+			goodAt: [""],
+		},
+	]);
+
+	const [areaForm, setAreaForm] = useState({
+		message: "",
+		light: "",
+	});
+
+	const handleSubmit = (e) => {
```
Comment (mnashawati) [inline client/src/components/FormAreasOfFocus/index.js:18]: Please push the changes that we did together on the call.

File: client/src/components/FormAreasOfFocus/index.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+			<div className="input-lights-add-button-container">
+				<div className="area-input-section">
+					<input
+						className="areas-input"
+						type="text"
+						name="area"
+						placeholder="eg. CSS, useEffect Hook, for loop..."
+						onChange={(e) =>
+							setAreaForm({
+								...areaForm,
```
Comment (mnashawati) [inline client/src/components/FormAreasOfFocus/index.js:40]: For this setState function as well as the following two, please push the changes from today's call before the merge.

**Review by mnashawati — CHANGES_REQUESTED** [review]
It looks very tidy, great job Ola! Just the main thing is to push the changes to "FormAreasOfFocus" from today's call before we merge it. Thank you!

**Review by mnashawati — APPROVED** [review]
Looks great!

File: client/src/components/StudentProfile/styles.css [hunk 6]
```diff
@@ -1,7 +1,8 @@
-.studen-profile-container {
+.student-profile-container {
     min-height: calc(100vh - 100px);
     display: flex;
     margin: 50px auto;
+    width:  960px;
```
Comment (mnashawati) [inline client/src/components/StudentProfile/styles.css:5]: This one should be max-width, but easily fixable later when we go through all the CSS

**Review by mnashawati — APPROVED** [review]
Great work!

**Comment by mnashawati** [comment]
Great stuff Ola!

**Pull request merged**